nextflow_schema.json

```diff
@@ -2,32 +2,43 @@
     "$schema": "http://json-schema.org/draft-07/schema",
     "$id": "https://raw.githubusercontent.com/nf-core/marsseq/master/nextflow_schema.json",
     "title": "nf-core/marsseq pipeline parameters",
-    "description": "MARS-seq 1/2 preprocessing pipeline",
+    "description": "MARS-seq v2 preprocessing pipeline",
```

Comment:
v2?

Reply:
ok, I follow, MARS-seq is the technology, you're using v2 in this pipeline which has the same name, but is in v1.
That's it?

Reply:
MARS-seq is an NGS technology, correct. First there was MARS-seq or v1, which had shorter cell and UMI barcodes. Then they came up with v2, which had longer ones. Since that was the case they released also pipeline MARS-seq2.0. And this is the one I've re-written. The original release v0.1 supported previously v1, but then I found out that the barcodes are shorter, so instead of rewriting some parts of the code where I work with this information I just dropped it and only kept v2.